smartsnippets/templates/smartsnippets/change_form.html

```diff
@@ -16,9 +16,9 @@
 {% endblock %}
 
 {% block extrastyle %}{{ block.super }}
-<link rel="stylesheet" type="text/css" href="{% admin_media_prefix %}css/forms.css" />
-<link rel="stylesheet" type="text/css" href="{% admin_media_prefix %}css/codemirror.css" />
-<link rel="stylesheet" type="text/css" href="{% admin_media_prefix %}css/smartsnippets-extra.css" />
+<link rel="stylesheet" type="text/css" href="{{ STATIC_URL }}admin/css/forms.css" />
+<link rel="stylesheet" type="text/css" href="{{ STATIC_URL }}admin/css/codemirror.css" />
+<link rel="stylesheet" type="text/css" href="{{ STATIC_URL }}admin/css/smartsnippets-extra.css" />
```

Comment:
Any reason not to use the `static` template tag here? https://docs.djangoproject.com/en/1.7/ref/contrib/staticfiles/#std:templatetag-staticfiles-static 

Could be good if we want to change our static file deployment strategy.